src/material/button-toggle/button-toggle.md

```diff
@@ -23,8 +23,9 @@ be configured globally using the `MAT_BUTTON_TOGGLE_DEFAULT_OPTIONS` injection t
 <!-- example(button-toggle-appearance) -->
 
 ### Accessibility
-The button-toggles will present themselves as either checkboxes or radio-buttons based on the
-presence of the `multiple` attribute.
+The button-toggles internally use native `button` elements with `aria-pressed` to convey
+their toggle state. The button-toggle-group surrounding the individual buttons applies
```

Comment:
Shouldn't it be "their toggle**d** state"?

Reply:
IMO both work in this context, but I changed it to "toggled" since if it were a checkbox I would indeed say "checked" and not "check".